osu.Game/Screens/Multiplayer/MultiUserPanel.cs

```diff
@@ -0,0 +1,191 @@
+                                {
+                                    Masking = true,
+                                    CornerRadius = 5,
+                                    Width = 30,
+                                    Height = 20,
+                                    FlagName = user.Country.FlagName,
+                                },
+                                new Container
+                                {
+                                    Masking = true,
+                                    CornerRadius = 5,
+                                    Width = 40,
+                                    Height = 20,
+                                    Children = new Drawable[]
+                                    {
+                                        new Box
+                                        {
+                                            RelativeSizeAxes = Axes.Both,
+                                            Colour = Color4.Gray,
+                                        }
+                                    }
+                                },
+                                new Container //osu! supporter tag
+                                {
+                                    Masking = true,
+                                    CornerRadius = 10,
+                                    Width = 20,
+                                    Height = 20,
+                                    Children = new Drawable[]
+                                    {
+                                        new Box
+                                        {
+                                            RelativeSizeAxes = Axes.Both,
+                                            Colour = Color4.Gray,
+                                        }
+                                    }
+                                },
+                            }
+                        }
+                    }
+                },
+                new CircularContainer
+                {
+                    Masking = true,
+                    Anchor = Anchor.BottomLeft,
+                    Origin = Anchor.BottomLeft,
+                    Size = new Vector2(12, 12),
+                    BorderColour = Color4.White,
+                    BorderThickness = 2,
+                    Colour = Color4.White,
+                    Margin = new MarginPadding { Left = 60, Bottom = 11 },
+                    Children = new[]
+                    {
+                        new Box
+                        {
+                            RelativeSizeAxes = Axes.Both,
+                            Alpha = 0,
+                            AlwaysPresent = true,
+                        },
+                    },
+                },
+                statusSprite = new OsuSpriteText
+                {
+                    Anchor = Anchor.BottomLeft,
+                    Origin = Anchor.BottomLeft,
+                    Text = "Waiting in a Room",
```

Comment:
Also casing here. Consider making these text pieces into private consts for now (will be localised later).